### ISSUE TITLE
[Feature Request] Suggestions After I am navigated to a New Thought Page from the New Thought Dialog

### ISSUE THREAD

**Issue by harshitpthk** [n3xah/issues#20]
After I am navigated to the New Chat Pane from the New Thought Dialog, I see the suggestions that may not be relevant to the thought that I have for e.g. 

The thought that I am pondering over is whether should I invest in Real Estate vs Dividend Stocks but the suggestions may not be relevant for that. Its subtle but it diverts your core attention from your thinking.

![image](https://github.com/user-attachments/assets/0cc7a61c-7f7b-4dc3-8331-a7a0dfc92e1f)

**Comment by harshitpthk** [comment]
I believe Chat About thought is present as an action but since suggestions appear first it might be helpful to do a suggestion based on the thought itself. 

![image](https://github.com/user-attachments/assets/803cc861-65c5-4aa7-8392-72c5d4964f29)

**Comment by brentbahry** [comment]
Yeah I think that's a good insight. Relevant chat actions are important. And I can see how it's distracting and lame when they're generic.
